types/src/on_chain_config/execution_config.rs

```diff
@@ -103,13 +103,13 @@ impl OnChainExecutionConfig {
 impl BlockGasLimitType {
     pub fn default_for_genesis() -> Self {
         BlockGasLimitType::ComplexLimitV1 {
-            effective_block_gas_limit: 30000,
+            effective_block_gas_limit: 20000,
```

Comment:
this also requires a governance proposal to update on mainnet? maybe we do it later after all changed calibrated

Reply:
yes, we can do it later.